src/files.js

```diff
@@ -74,7 +82,15 @@ function doUpload(cozy, data, method, path, options) {
   if (lastModifiedDate) headers['Date'] = lastModifiedDate.toGMTString()
   if (ifMatch) headers['If-Match'] = ifMatch
 
-  return cozyFetch(cozy, path, {
+  let finalpath = path
```

Comment:
Why are we introducing a new `finalPath` variable?

Reply:
I introduced it because I don't like to change function parameters.

Reply:
I second that. Changing function parameters is not really a recommended pattern